contrib/sqlalchemy-connector/sqlalchemy_starrocks/dialect.py

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/python3
+# This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+from sqlalchemy.dialects.mysql import BIGINT
+from sqlalchemy.dialects.mysql.mysqldb import MySQLDialect_mysqldb
+
+class StarRocksDialect(MySQLDialect_mysqldb):
+
+    name = 'starrocks'
+
+    def __init__(self, *args, **kw):
+        super(StarRocksDialect, self).__init__(*args, **kw)
+        self.ischema_names = self.ischema_names.copy()
+        self.ischema_names['largeint']=BIGINT
```

Comment:
I don't know whether this conversion is feasible or not. I may need to test whether it is better to convert LARGEINT to BIGINT or VARCHAR？

Reply:
After consideration, convert LARGEINT to DECIMAL.